packages/headless/src/app/engine-configuration.ts

```diff
@@ -110,11 +110,15 @@ export interface AnalyticsConfiguration {
    */
   originContext?: string;
   /**
-   * Origin level 2 is a usage analytics event metadata whose value should typically be the name/identifier of the tab from which the usage analytics event originates.
+   * Sets the value of the Origin Level 2 dimension on the analytic events.
```

Comment:
```suggestion
   * Sets the value of the Origin Level 2 dimension on the analytics events.
```

I see another instance of it in the originContext docs if you want to fix it as well 😄